SampleApp/SampleApp/MessagesViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  MessagesViewController.swift
+//  SampleApp
+//
+//  Created by Peter Livesey on 7/22/16.
+//  Copyright © 2016 LinkedIn. All rights reserved.
+//
+
+import UIKit
+import RocketData
+
+class MessagesViewController: UIViewController, CollectionDataProviderDelegate, DataProviderDelegate, UITableViewDataSource, UITableViewDelegate {
```

Comment:
I would add some class doc as well as inline doc in viewDidLoad to help readers understand what is happening